go.mod

```diff
@@ -1,23 +1,50 @@
 module github.com/zlabjp/spire-openstack-plugin
 
-go 1.12
+go 1.13
```

Comment:
Should we update the following files depending on this change? What do you think?

- https://github.com/zlabjp/spire-openstack-plugin/blob/7e9b05946219a79ab72051f80214340cbea9bd7a/.travis.yml#L4
- https://github.com/zlabjp/spire-openstack-plugin/blob/a4686dc3854bc86a3abef3ee8c92d9a9fb97ac4d/CONTRIBUTING.md#prerequisites